package.json

```diff
@@ -10,16 +10,23 @@
   "bugs": {
     "url": "https://github.com/alveusgg/extension/issues"
   },
+  "engines": {
+    "node": "22.x",
+    "pnpm": "^9.14.2"
+  },
+  "packageManager": "pnpm@9.14.2",
   "scripts": {
     "prepare": "husky",
     "dev": "cross-env NODE_ENV=development webpack serve --config src/webpack.config.ts",
     "build": "webpack --config src/webpack.config.ts",
-    "lint:eslint": "eslint 'src/**/*.{js,jsx,ts,tsx}'",
+    "lint:eslint": "eslint \"src/**/*.{js,jsx,ts,tsx}\"",
     "lint:prettier": "prettier --check \"src/**/*.{js,jsx,ts,tsx,css,json,html}\"",
     "lint": "run-p lint:*",
-    "format:eslint": "npm run lint:eslint -- --fix",
-    "format:prettier": "npm run lint:prettier -- --write",
-    "format": "run-p format:*"
+    "format:eslint": "pnpm lint:eslint --fix",
+    "format:prettier": "pnpm lint:prettier --write",
+    "format": "run-p format:*",
+    "types": "tsc --noEmit",
+    "clean": "rm -rf build node_modules"
```

Comment:
Why do we need a clean command (and one that I don't think will work on Windows)?